docker-compose.yml

```diff
@@ -11,7 +11,7 @@ services:
     depends_on:
       - mongo
   mongo:
-    image: mongo:3.4.2
+    image: mongo:4.4
```

Comment:
Are we dropping support to Mongo below 4.4?

We need proper changelog details.